client/plugin_install.go

```diff
@@ -65,7 +74,12 @@ func (cli *Client) PluginInstall(ctx context.Context, name string, options types
 	return cli.PluginEnable(ctx, name, types.PluginEnableOptions{Timeout: 0})
 }
 
-func (cli *Client) tryPluginPull(ctx context.Context, query url.Values, registryAuth string) (serverResponse, error) {
+func (cli *Client) tryPluginGetPrivileges(ctx context.Context, query url.Values, registryAuth string) (serverResponse, error) {
```

Comment:
nit: can be `tryPluginPrivileges` to be consistent with other client function names.